locale/fr.po

```diff
@@ -20,132 +20,123 @@ msgstr ""
 
 #: init.lua
 msgid "Air (you hacker you!)"
-msgstr ""
+msgstr "Air (espèce de tricheur ! )"
```

Comment:
Sorry but it's a french typographic rule : some punctuation signs (colon, exclamation mark, question mark, semicolon, some guillemets) need to be preceded and followed by a space.